### ISSUE TITLE
Add Freising

### ISSUE THREAD

**Issue by lukf** [EqualStreetNames/equalstreetnames#157]
I would like to transfer my Freising repo to the EqualStreetNames account.

https://github.com/lukf/equalstreetnames-freising

**Comment by jbelien** [comment]
Hello @lukf , that's an EXCELLENT news ! 🎉

- You need to go to your repository settings : <https://github.com/lukf/equalstreetnames-freising/settings>
- Click on the "Transfer" button in the "Danger Zone"
- Put "EqualStreetNames" in the "New owner’s GitHub username or organization name" field
- Type your repository name to confirm
- And transfer it

**Comment by lukf** [comment]
@jbelien when I attempt the transfer, it fails with "You don’t have the permission to create public repositories on EqualStreetNames"

**Comment by jbelien** [comment]
Ha ! Interesting ! 😄 
I probably need to create your team first.
I'll do that later today and come back to you ! 👍

**Comment by lukf** [comment]
After joining the EqualStreetNames GitHub team, transfer was successful. Thanks!

**Comment by jbelien** [comment]
Awesome !
Welcome on board ! 🚀 

You now have admin rights on `EqualStreetNames/equalstreetnames-freising` repository and are maintainer of the "Freising" team.

I'll setup the automated data update and deployment later today.

**Comment by jbelien** [comment]
I have added the sub-module to the main repository and set up the automated data update process.

For a reason that I can't explain (for now), the Overpass query do not return anything when I run the process ...
I'll investigate !

**Comment by lukf** [comment]
Maybe we are getting outdated data. I added all the Wikidata tags to OSM yesterday.

**Comment by jbelien** [comment]
No, sorry, it was an issue with the Overpass queries from the template.
I fixed it ; data are now available here : https://github.com/EqualStreetNames/equalstreetnames-freising/tree/master/data

I'll deploy the website later this week.

**Comment by jbelien** [comment]
Lift off : https://freising.equalstreetnames.eu/ 🚀
